release_files/Dockerfile.ubuntu

```diff
@@ -163,6 +167,13 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
 # for NCS
 RUN if [ -f /ovms/lib/hddl/etc/rules.d/97-myriad-usbboot.rules ]; then mkdir -p /etc/udev/rules.d/ && cp /ovms/lib/hddl/etc/rules.d/97-myriad-usbboot.rules /etc/udev/rules.d/ && ldconfig ; fi
+# for NVIDIA
+RUN if [ "$NVIDIA" == "1" ]; then true ; else exit 0 ; fi ; echo "installing cuda apt package"; \
+    apt update -y && \
```

Comment:
We use curl in #43 
(So I expect it is already present in OS)

Reply:
it is a difference image